### ISSUE TITLE
Buttons in grid WidgetColumn cells cannot be invoked using keyboard

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#9390]
Should react to SPACE + ENTER keys

http://lh/bryntum-suite/grid/examples/columntypes/
